app/models/flexible_criterion.rb

```diff
@@ -100,6 +100,16 @@ def get_weight
     1
   end
 
+  # Returns the value entered when the criterion is set up.
+  def mark_set_up_value
+    max_mark
+  end
+
+  # Returns the maximum mark for a particular criterion.
+  def max_mark
```

Comment:
For now, the value returned by mark_set_up_value and max_mark is the same here in Flexible Criterion, but it is different in Rubric Criterion. I will change the name, add a TODO comment and get rid of the method in my next pull request, because I am using both methods in assignment.rb right now. I hope this is ok with you.